doc/CodeStyle

```diff
@@ -20,8 +20,10 @@
  - an output argument which is a pointer to a user variable has _p suffix
  - value types (e.g struct types, integer types) have _t suffix
  - pointer to structs, which are used as API handles, have _h suffix
- - macro arguments begine with _ (e.g _value) to avoid confusion with variables
+ - macro arguments begin with _ (e.g _value) to avoid confusion with variables
  - no leading underscores in function names
+ - header file names must use _fwd.h suffix if contains forward declarations
```

Comment:
no. forward declarations are _fwd.h type declarations are _types.h. marco/preprocessor are _def.h. inline functions are .inl
for clarification, a forward declaration is this:
`typedef struct opaque_struct opaque_struct_t"
or this:
`vo my_func(int foo)"